catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -614,19 +614,20 @@ public boolean isForcedAutoCommit() {
     }
 
     public void shutdown() {
-        LOGGER.info("Shutting down solr server.");
-        server.shutdown();
+        LOGGER.info("Shutting down Solr server.");
+        try {
+            server.close();
+        } catch (IOException e) {
+            LOGGER.info("Failed to shutdown Solr server.", e);
+        }
     }
 
     private class ProviderSolrMetacardClient extends SolrMetacardClient {
 
-        public ProviderSolrMetacardClient(SolrServer solrServer, FilterAdapter catalogFilterAdapter,
+        public ProviderSolrMetacardClient(SolrClient client, FilterAdapter catalogFilterAdapter,
                 SolrFilterDelegateFactory solrFilterDelegateFactory,
                 DynamicSchemaResolver dynamicSchemaResolver) {
-            super(solrServer,
-                    catalogFilterAdapter,
-                    solrFilterDelegateFactory,
-                    dynamicSchemaResolver);
+            super(client, catalogFilterAdapter, solrFilterDelegateFactory, dynamicSchemaResolver);
```

Comment:
That is with the correct formatter.  THis line is now under the 100 character line limit because I renamed `solrServer` to `client`.